R/PipeOpBreslow.R

```diff
@@ -0,0 +1,119 @@
+#' @title PipeOpBreslow
+#' @name mlr_pipeops_compose_breslow_distr
+#' @description
+#' A short description...
+#'
+PipeOpBreslow = R6Class("PipeOpBreslow",
+  inherit = mlr3pipelines::PipeOp,
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    #' @param id description
+    initialize = function(learner, id = "po_breslow", param_vals = list(overwrite = FALSE)) {
+      if ("lp" %nin% learner$predict_types) {
+        stopf("Learner %s must provide lp predictions", learner$id)
+      }
+
+      private$.learner = as_learner(learner, clone = TRUE)
+
+      # PipeOp-specific ParamSet
+      ps_pipeop = ps(
+        overwrite = p_lgl(default = FALSE, tags = c("predict"))
+      )
+      # add learner's ParamSet
+      param_set = ParamSetCollection$new(
+        list(private$.learner$param_set, ps_pipeop)
+      )
+
+      super$initialize(
+        id = id,
+        param_set = param_set,
+        param_vals = param_vals,
+        input = data.table(name = "input", train = "TaskSurv", predict = "TaskSurv"),
+        output = data.table(name = "output", train = "NULL", predict = "PredictionSurv"),
+        packages = learner$packages
+      )
+    }
+  ),
+
+  active = list(
+    learner = function(rhs) {
+      assert_ro_binding(rhs)
+      private$.learner
+    }
+  ),
+
+  private = list(
+    .train = function(inputs) {
+      # browser()
+      task = inputs[[1]]
+      learner = private$.learner
```

Comment:
I don't understand this syntax, why aren't you using inputs for the learner like all other pipeops?

Reply:
Discussion in Mattermost...